### ISSUE TITLE
nice but could use some features. An unsolicited review based on my attempt to get it working.

### ISSUE THREAD

**Issue by ksdavidc** [nhannht/obsidian-historica#16]
I like the approach as it makes it very easy to set up the timeline. Easiest of all the timeline plugins I've seen, especially since the data is still in markdown. (most of them require codeblocks)

However:

- tried using dataview or other query to pull in data, but that didn't work.
- additional control over styling would be nice, or at least a list of css elements to target.
- for example, it seems to include the date in 3 places, the timeline, the title of the entry, and the entry itself. seems like overkill, unless I am missing somethign.
- when editing an entry the text editor text area field supports various styles, like bold, etc, but doesn't appear to be markdown based, for example it doesn't seem to include checkboxes. Plugin saves are in json format. By sticking to markdown, edits it would make it easier for you to apply edits to the original blocks, or export to new markdown.
-  I ended up reading the readme cursorily and only after playing around for a while did I understand how to save and how to sort and that right clicking different parts of the timeline show different context menus. I should have read it carefully, though it could use a brush up...
- likewise, only then did I see how to add more files, and it seems I have to add them individually. It would be nice to have a search function, so I could for example get all files with a certain name, or tag, etc.
- I would like to know what formats of dates it understands, though I discovered it seems to handle natural language dates like "a month ago". This is fine, but words like now, which are too common, get caught too..
- likewise the parse function surprised me in its behavior. I see now it adds a new timeline element, not replaces the current. That is probably better, but I misread it.

Now that I sorted through all that, I think I know how I could use it certain circumstances, but I think it needs some updating to be more generally useful.

**Comment by ksdavidc** [comment]
don't feel any obligation to do what I suggested. Just my observations in trying to get it working for me. really would love a markdown export...
